src/components/layouts/FilterBar.tsx

```diff
@@ -0,0 +1,34 @@
+'use client'
+
+import { Box, Stack, Typography } from '@mui/material'
+import { AppMargin, SizeofAppMargin } from '@/hoc/AppMargin'
+import { useEffect, useState } from 'react'
+import store from '@/redux/store'
+import { setFilteredTasks } from '@/redux/features/taskBoardSlice'
+import SearchBar from '@/components/searchBar'
+
+export const FilterBar = ({}: {}) => {
+  const [searchText, setSearchText] = useState('')
+
+  useEffect(() => {}, [searchText])
```

Comment:
Why do we have an empty effect here? Is this to trigger a filter in the future?

If it's not required please remove it